SMART Health Cards.ipynb

```diff
@@ -520,23 +508,48 @@
     }
    ],
    "source": [
+    "// Create array of chunks to convert into numeric mode QRs\n",
     "var QRCode = require('qrcode')\n",
-    "\n",
-    "const segments = [\n",
+    "const jwsChunks = splitJwsIntoChunks(jws);\n",
+    "\n",
+    "// Save QR code data for future comparison with scanned QR Code\n",
+    "let qrCodesData = [];\n",
+    "\n",
+    "jwsChunks.forEach(async function(jwsChunk, i) {\n",
+    "    // This code snippet was adapted/taken from https://github.com/smart-on-fhir/health-cards/blob/152a4f83b223b5fd14027f765e957e290649f2c0/generate-examples/src/index.ts#L173\n",
+    "    const numericJWS = await jws.split('')\n",
+    "        .map((c) => c.charCodeAt(0) - 45)\n",
+    "        .flatMap((c) => [Math.floor(c / 10), c % 10]) //Need to maintain leading zeros\n",
+    "        .join('')\n",
+    "\n",
+    "    const qrCodeData = 'shc:/' + numericJWS\n",
+    "    \n",
+    "    qrCodesData.push(qrCodeData);\n",
+    "    \n",
+    "    const segments = [\n",
     "    {data: 'shc:/', mode: 'byte'},\n",
     "    {data: numericJWS, mode: 'numeric'}\n",
-    "]\n",
-    "\n",
-    "let qrSVG\n",
-    "QRCode.toString(segments,{type: 'svg', errorCorrectionLevel: 'low', version: 22 }).then(function(result){\n",
-    "    qrSVG = result\n",
-    "    $$.svg(qrSVG)\n",
-    "})\n",
+    "    ]\n",
```

Comment:
This is missing code to add the index of the current chunk and total chunks.

These won't be present with N=1 QR codes anyways, but for informational purposes we should include code for it.

Relevant spec section: https://spec.smarthealth.cards/#encoding-chunks-as-qr-codes

Example Code: https://github.com/smart-on-fhir/health-cards/blob/9e58355544f0203fe46bc92df0e42ac4d890f1f9/generate-examples/src/index.ts#L211

![Screen Shot 2022-02-02 at 10 14 34 AM](https://user-images.githubusercontent.com/41651655/152182448-f4cab1fc-373c-4f5e-93e3-8326ebf884bc.png)

Reply:
Also `const qrCodeData = 'shc:/' + numericJWS` should be removed entirely and we should be pushing the result of `await QRCode.toString...` onto `qrCodesData` instead.